packages/flutter_tools/lib/runner.dart

```diff
@@ -75,22 +75,18 @@ Future<int> _handleToolError(
     String getFlutterVersion(),
     ) async {
   if (error is UsageException) {
-    stderr.writeln(error.message);
-    stderr.writeln();
-    stderr.writeln(
+    printError('${error.message}\n');
+    printError(
         "Run 'flutter -h' (or 'flutter <command> -h') for available "
             'flutter commands and options.'
```

Comment:
please fix the crazy indentation on this line while you're here

Reply:
Done.